src/ControlComponents/DynamicpositioningLock.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Switch from './Switch';
+import Title from './Title';
+import './css/NetfollowingLock.css';
+
+const { remote } = window.require('electron');
+
+export default function DynamicpositioningLock({ title, active, step }) {
+  const [input, changeInput] = useState(0.0);
```

Comment:
Here we should use the same principle as in NetfollowingLock with 3 different inputs so we don't get the bug where the last number changed is the one getting updated.